blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/AstyanaxReader.java

```diff
@@ -679,4 +679,32 @@ public Boolean apply(Row<Locator, Long> row) {
         }
         return columns;
     }
+
+    public Map<Locator, ArrayList<String>> getEnumStringMappings(final List<Locator> locators) {
```

Comment:
@tilogaat I added this method which will query return locator to ArrayList of enum string values mapping, please review.